ledger/metrics/src/main/scala/com/daml/metrics/HttpJsonApiMetrics.scala

```diff
@@ -25,43 +30,78 @@ class HttpJsonApiMetrics(val prefix: MetricName, override val registry: MetricRe
 
   val surrogateTemplateIdCache = new CacheMetrics(prefix :+ "surrogate_tpid_cache", registry)
 
+  object DropWizardMetricsFactory extends DropwizardFactory {
```

Comment:
For easier readability, I'd suggest declaring a `val` instead named like `metricsFactory`.